Frameworks/CoreText/CTFont.mm

```diff
@@ -415,21 +442,27 @@ CFArrayRef CTFontCopySupportedLanguages(CTFontRef font) {
     return StubReturn();
 }
 
+// Private helper for getting a DWRITE_FONT_METRICS struct, to avoid code duplication
+static struct DWRITE_FONT_METRICS __CTFontGetDWriteMetrics(CTFontRef font) {
+    struct DWRITE_FONT_METRICS metrics;
+    font->_dwriteFont->GetMetrics(&metrics);
```

Comment:
yeah, I think so

---
In reply to: [78206815](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78206815) [](ancestors = 78206815)